Misc/NEWS.d/next/Library/2024-05-08-18-33-07.gh-issue-118507.OCQsAY.rst

```diff
@@ -1 +1,2 @@
+Fix :func:`os.path.isfile` on Windows for pipes.
```

Comment:
This is a substantive change that doesn't match what the PR title says this PR does.

That might be fine, but it means I'm going to refrain from accepting this PR, since I don't have the context to evaluate this substantive change.

Reply:
This was only mentioned in the backport to 3.12, I've moved this to a separate PR.